### PR TITLE
Update Gluten + HDFS docs.

### PR DESCRIPTION
**Pull request by Yohahaha**
## What changes were proposed in this pull request?

Update docs since #788 changes.

### CODE REVIEW

**Comment by Yohahaha** [comment]
Hi @zhouyuan, could you help me review this docs? Thanks!

File: docs/Velox.md [hunk 1]
```diff
@@ -129,18 +129,32 @@ cd /path_to_gluten/cpp
 cd /path_to_gluten
 mvn clean package -Pbackends-velox -Pspark-3.2 -Pfull-scala-compiler -DskipTests -Dcheckstyle.skip
 ```
-Gluten HDFS support requires an extra environment variable "VELOX_HDFS" to indicate the Hdfs URI. e.g. VELOX_HDFS="host:port". If the env variable is missing, Gluten will try to connect with hdfs://localhost:9000
+Gluten HDFS support requires Hdfs URI, you can define this config in three ways:
 
-This env should be exported in both Spark driver and worker.
-e.g., in Spark local mode:
+1. spark-defaults.xml
```
Comment (zhouyuan) [inline docs/Velox.md:134]: I think you meant for `spark-defaults.conf`

**Comment by zhouyuan** [comment]
@Yohahaha thanks for the follow up!

File: docs/Velox.md (hunk 1 above)
Comment (Yohahaha) [reply docs/Velox.md]: already fix this.

**Review by zhztheplayer — APPROVED** [review]

**Pull request merged**